src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -16,9 +16,8 @@
  * Provide default OG selection handler.
  *
  * @EntityReferenceSelection(
- *   id = "default:og",
+ *   id = "og:default",
```

Comment:
The idea is that we will take the original query object provided by the original selection handler(could be for node, term, user or even commerce product) and apply OG additional logic.